VAMobile/src/screens/SyncScreen/SyncScreen.tsx

```diff
@@ -95,16 +99,31 @@ function SyncScreen({}: SyncScreenProps) {
 
   return (
     <VAScrollView {...testIdProps('Sync-page')} contentContainerStyle={splashStyles} removeInsets={true}>
+      <StatusBar
+        translucent
+        barStyle={theme.mode === 'dark' ? 'light-content' : 'dark-content'}
+        backgroundColor={theme.colors.background.main}
+      />
       <Box
         justifyContent="center"
         mx={isPortrait ? theme.dimensions.gutter : theme.dimensions.headerHeight}
         mt={theme.dimensions.contentMarginTop}
         mb={theme.dimensions.contentMarginBottom}
         alignItems={'center'}>
-        <VAIcon name={'Logo'} />
+        <Image
+          style={logoProps}
+          source={
```

Comment:
i'm open to a better solution for the source here, it's escaping me.

Reply:
Off the top of my head you could do a exported constant or an env variable and import it

Reply:
Exporting a const for this would make it easier to maintain. You could even create a helper function for returning the icon Image with the required props and everything, that way you don't have to set it in multiple places. On a similar note, since this is used in 3 different places, you could even make a component for this, e.g. `VALogo`. That way, you could reference `<VALogo />` wherever you need it.

Also, you should be able to import `@department-of-veterans-affairs/mobile-assets/VALogo/VAOnDark.png`, you shouldn't need to specify the relative path